src/components/RestaurantDetails/RestaurantDetails.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable jsx-a11y/img-redundant-alt */
+//Libraries
+import React from 'react';
+
+//Helpers
+import fetchData from '../../api/fetchData';
+import restaurantDetailsReader from './readers/restaurantDetailsReader';
+
+//Icons
+import star from '../../images/star.svg';
+import delivery from '../../images/delivery.svg';
+import rupee from '../../images/rupee.svg';
+
+//Constants
+import {
+  INITIAL_STATE,
+  RESTAURANT_ID,
+} from './constants/restaurantDetails.general';
+
+//Components
+import Offers from './components/Offers';
+import Filters from './components/Filters';
+import RestaurantInfo from './components/RestaurantInfo';
+import AdditionalInfo from './components/AdditionalInfo';
+import Loader from '../../commonComponents/Loader';
+
+//CSS
+import './RestaurantDetails.css';
+
+class RestaurantDetails extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = INITIAL_STATE;
+    this.additionalInfoProps = this.additionalInfoProps.bind(this);
```

Comment:
try to follow one pattern, either bind the callback fns or instead use arrow fns